cluster/images/etcd/start-stop-etcd.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/sh
+
+# Copyright 2016 The Kubernetes Authors.
```

Comment:
Moved these functions to their own source file so they can be leveraged by the tests.